common/changes/@microsoft/rush/rename-task-to-operation_2022-02-08-02-08.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Rename \"Task\" to \"Operation\" in source code to conform to documentation.",
```

```suggestion
      "comment": "",
```

A user or API consumer isn't going to be affected by this change.